server/src/main/java/io/deephaven/server/barrage/BarrageMessageProducer.java

```diff
@@ -998,6 +997,41 @@ public void scheduleAt(final long nextRunTime) {
         }
     }
 
+    /**
+     * @formatter:off
```

Comment:
Given that you've added this as a JavaDoc comment, it would be more appropriate to leave the formatter on and use HTML to format your text.